example/src/lib.rs

```diff
@@ -11,54 +11,73 @@ mod ffi {
     use writeable::Writeable;
 
     #[diplomat::opaque]
+    /// A decimal number. See [the Rust docs](https://unicode-org.github.io/icu4x-docs/doc/fixed_decimal/decimal/struct.FixedDecimal.html) for more information.
     pub struct ICU4XFixedDecimal(pub FixedDecimal);
 
     impl ICU4XFixedDecimal {
+        /// Construct an [`ICU4XFixedDecimal`] from an integer.
```

Comment:
issue: you'd want this link to also be fully specified, since we don't have a way to do intra-doc yet (and it's going to be hard to do that well)

Reply:
Or make it a non-link, we don't actually need a link here

Reply:
I was thinking we could attempt to resolve this by first looking at the set of custom types in the module, and then error out if it's not found (eventually replaced with actual intra-doc logic). So then for local references you wouldn't need full paths.